bindep_combined.txt

```diff
@@ -16,3 +16,4 @@ qemu-img [platform:rpm]  # from collection ovirt.ovirt
 python3-devel [platform:rpm compile]  # from collection user
 subversion [platform:rpm]  # from collection user
 subversion [platform:dpkg]  # from collection user
+glibc-langpack-en  # from collection user
```

Comment:
would it be better to mark this with `[platform:rpm]`? I would look to @pabelanger for the answer

Reply:
yup, you are right. Will fix.